### PR TITLE
anglarjs developer materials - DEVELOPER-1135

### PR DESCRIPTION
**Pull request by wesbos**
WIP Angular for review - chat on IRC

### CODE REVIEW

**Comment by paulrobinson** [comment]
retest this please

**Comment by wesbos** [comment]
retest this please

**Comment by paulrobinson** [comment]
retest this please

File: _layouts/base.html.slim [hunk 1]
```diff
@@ -197,7 +198,10 @@ body(class="#{page.url.gsub('/','').split('.').first || 'home'} #{page.body_clas
     / Now load anything that is going to query the DCP
     script src="#{site.base_url}/javascripts/dcp.js"
     script src="#{site.base_url}/javascripts/buzz.js"
-    script src="#{site.base_url}/javascripts/developer-materials.js"
+    / script src="#{site.base_url}/javascripts/developer-materials.js"
```
Comment (paulrobinson) [inline _layouts/base.html.slim]: can this be removed?

File: _picketlink-quickstarts [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit 81554884f82c38226973b18b682e4c10ff957298
+Subproject commit 2c849738e14bd9ee7ccd42c663d3b5ec1540c867
```
Comment (paulrobinson) [inline _picketlink-quickstarts]: This needs removing

**Comment by paulrobinson** [comment]
Vineet is reviewing the code for this.

**Comment by paulrobinson** [comment]
Target product is now a list, so this column needs updating: https://www.dropbox.com/s/ihy7sdkq89fv7il/Screenshot%202015-02-04%2010.31.21.png?dl=0

**Comment by paulrobinson** [comment]
I've fixed the small issues I mentioned above and raised: https://github.com/jboss-developer/www.jboss.org/pull/800

**Pull request closed without merging**